### ISSUE TITLE
Need help for connecting rabbitmq from AmazonMQ

### ISSUE THREAD

**Issue by prateek9623** [alanxz/SimpleAmqpClient#294]
Hi,
I have a rabbitmq instance running in AWS using AmazonMQ to which I am trying to connect. But facing an issue while connecting, after some digging up, I found out that to create a secure connection I have to pass the CA cert, client cert, and client pass paths. 
But I don't know how to get those. Can anyone help me?

**Comment by alanxz** [comment]
I don't have a lot of experience with AmazonMQ.   If you can find documentation on how to correctly configure a client to connect to a TLS-enabled AmazonMQ broker, then I can help translate those requirements into how to use SimpleAmqpClient

**Comment by prateek9623** [comment]
AmazonMQ is simply just managed RabbitMQ where all cluster management and security setup is handled by AWS.
And I am using this broker in 2 of my application, in first of which is written in Python and using aio_pika and I don't face any issue there, its code looks something like this: 
```python
from aio_pika import connect_robust
connection = connect_robust('amqps://user:pass@uniqueID.mq.region.amazonaws.com:5671' )
```
note here I don't need to send any certificate here

but to do a similar thing in SimpleAmqpClient, I think I need to assign cert path in:
TLSParams:client_key_path, TLSParams:client_cert_path, TLSParams:ca_cert_path but i don't know how to get that.

Also, while exploring rabbitmq-c, I found
https://github.com/alanxz/rabbitmq-c/blob/63c70b7b43c7fa73ae8ee9d528f4b3611065a16f/examples/amqp_ssl_connect.c
and I tried this example with just URL and I was successful to connect, so I looked into example code and I found out at line 103 that cert is optional, so I tried to replicate that in SimpleAmqpClient

**Comment by alanxz** [comment]
Resolved by #309
